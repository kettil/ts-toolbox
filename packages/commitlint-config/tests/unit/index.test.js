const config = require('../../src');

describe('index', () => {
  test('it should work', () => {
    expect(config).toMatchSnapshot();
  });
});
