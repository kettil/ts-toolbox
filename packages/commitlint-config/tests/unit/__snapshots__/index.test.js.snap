// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index it should work 1`] = `
Object {
  "rules": Object {
    "body-leading-blank": Array [
      1,
      "always",
    ],
    "body-max-line-length": Array [
      2,
      "always",
      120,
    ],
    "footer-leading-blank": Array [
      1,
      "always",
    ],
    "footer-max-line-length": Array [
      2,
      "always",
      120,
    ],
    "header-max-length": Array [
      2,
      "always",
      72,
    ],
    "scope-case": Array [
      2,
      "always",
      "lower-case",
    ],
    "subject-case": Array [
      2,
      "never",
      Array [
        "sentence-case",
        "start-case",
        "pascal-case",
        "upper-case",
      ],
    ],
    "subject-empty": Array [
      2,
      "never",
    ],
    "subject-full-stop": Array [
      2,
      "never",
      ".",
    ],
    "type-case": Array [
      2,
      "always",
      "lower-case",
    ],
    "type-empty": Array [
      2,
      "never",
    ],
    "type-enum": Array [
      2,
      "always",
      Array [
        "build",
        "chore",
        "ci",
        "docs",
        "feat",
        "fix",
        "perf",
        "refactor",
        "revert",
        "style",
        "test",
      ],
    ],
  },
}
`;
